src/App.js

```diff
@@ -5,11 +5,19 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './PostList';
+
+const createPosts = posts.map(post => (
```

Comment:
```suggestion
const createdPosts = posts.map(post => (
```
createPosts - название для функции, потому что в названии есть действие - create